sdk/logical/logical.go

```diff
@@ -103,6 +103,10 @@ type BackendConfig struct {
 	// when a backend needs a consistent and unique string without using storage.
 	BackendUUID string
 
+	// ClusterUUID is a unique identifier for this Vault cluster. This is used
+	// with the path-based write forwarding mechanism.
+	ClusterUUID string
```

Comment:
@ncabatoff I've added a new `ClusterUUID` flag here in the top commit, let me know what you think.

Reply:
Seems fine.

Reply:
Actually, I wonder if instead of putting it here, we could add it as a SystemView method?  That seems like the more conventional way we would approach something like this.

Reply:
We could, but I think we'd have to decide if we wanted to support it on GRPC plugins? I started going down that route but realized it was a lot more work. Perhaps best left for someone more familiar with it. PKI isn't a GRPC client, so maybe it doesn't need to happen until then.

Reply:
The other comment I'd make is that the ClusterUUID is static throughout the lifetime of the mount. Presumably, if this info changes, the plugin would be entirely reloaded due to cluster status change (becoming a standby/...). Having timely info about this as the status was changing (e.g., if we were to become a perf standby) wouldn't necessarily help much IMO.  

But if you want it there, for non-GRPC plugins, I think its only a few more lines of code total, so I'm happy to do whatever.

Reply:
A lot more work how?  Isn't it just a question of a simple method in dynamicSystemView?  I've added systemview methods before and recall it being easy.

Reply:
Right, for dynamicSystemView its easy, but for GRPC non-builtin plugins, you've gotta add it to both GRPCClient + Server and I got lost there.

Reply:
@ncabatoff Pushed, LMKWYT.

Reply:
@ncabatoff Just pushed a GRPC implementation, not sure if it is correct :-D